examples/jsd.py

```diff
@@ -0,0 +1,342 @@
+"""
+Helion JSD (Jensen-Shannon Divergence) Example
+==============================================
+This example demonstrates a Helion kernel implementation of Jensen-Shannon Divergence.
+JSD is commonly used in knowledge distillation for language models, where:
+
+JSD(β)(P || Q) = β * KL(P || M) + (1-β) * KL(Q || M)
```

Comment:
maybe change `β` to `beta` in this file - a bit unsure how non-ASCII character will show up in all systems